helpers/auth/authentication.py

```diff
@@ -104,10 +105,15 @@ def wrapper(*args, **kwargs):
                     user = User.query.filter_by(email=email).first()
                     headers = {"Authorization": 'Bearer ' + self.get_token()}
                     try:
-                        data = requests.get(
-                            api_url + "users?email=%s"
-                            % user.email, headers=headers)
-                        response = json.loads(data.content.decode("utf-8"))
+                        if(os.getenv('APP_SETTINGS') == "testing"):
+                            with open('users.json', 'r') as f:
+                                response = json.load(f)
+                        elif(os.getenv('APP_SETTINGS') ==
+                                "development"):  # pragma: no cover
```

Comment:
@michael-basweti how about for `production`? What settings will they use?

Reply:
@joshuaocero I have taken care of it.